wger/weight/views.py

```diff
@@ -166,25 +167,28 @@ def get_weight_data(request, username=None):
     Process the data to pass it to the JS libraries to generate an SVG image
     '''
 
-    is_owner, user = check_access(request.user, username)
+    user = User.objects.filter(username=username)
 
     date_min = request.GET.get('date_min', False)
     date_max = request.GET.get('date_max', True)
 
     if date_min and date_max:
-        weights = WeightEntry.objects.filter(user=user,
-                                             date__range=(date_min, date_max))
+        weights1 = WeightEntry.objects.filter(user=request.user, date__range=(date_min, date_max))
```

Comment:
@Emmanuel-Dominic please consider renaming this variable as it is discouraged to use integers in variable names. Consider something more descriptive say `initial_weight` or `first_weight`

Reply:
This has been resolved.